src/index.js

```diff
@@ -1,7 +1,13 @@
 'use strict';
 
-var User = require('./models/user');
-var user = new User('root', 10);
+var request = require('./utils/request');
 
-console.assert(user.triage() === 30 && user.age === 30);
-console.assert(user.rename('markku') === 'markku' && user.name === 'markku');
+var unused_variable;
```

Comment:
Identifier 'unused_variable' is not in camel case.<br>'unused_variable' is defined but never used.